core/domain/blog_services_test.py

```diff
@@ -326,6 +343,16 @@ def test_update_blog_post(self) -> None:
         self.assertEqual(
             updated_blog_post.url_fragment, 'sample-title' + lower_id)
 
+        # Providing same title in change dict raises no error.
+        # Here we use MyPy ignore because dictionary of type BlogPostChangeDict
+        # should contain 'title', 'content', 'tags' and 'thumbnail_filename'
+        # keys but for testing purpose here we are only providing the 'title'
+        # key, which causes MyPy to throw an error. Thus to silent the error, we
```

Comment:
```suggestion
        # key, which causes MyPy to throw an error. Thus to silence the error, we
```

Sorry, I missed this before

Reply:
Done